docs/modules/chat.mdx

```diff
@@ -22,22 +36,28 @@ chat:
 [Vault](https://github.com/milkbowl/Vault) is required if you wish to retrieve prefixes or suffixes from a permission manager, or use group based formatting.
 :::
 
-These are the configurations for a global chat format. This is how your chat messages will be displayed when a player types in the server chat.
+These configurations define the global chat format, controlling the appearance of chat messages. You can use the
+following placeholders:
+
 ```yaml
   format: '<{DISPLAYNAME}> {MESSAGE}'
   #format: '&7[{GROUP}]&r {DISPLAYNAME}&7:&r {MESSAGE}'
   #format: '&7{PREFIX}&r {DISPLAYNAME}&r &7{SUFFIX}&r: {MESSAGE}'
 ```
 
-You can set formats per group in EssentialsX Chat. make sure that the name you use in this section matches exactly what you have in your permission manager
+EssentialsX Chat allows you to define different chat formats for each group. Ensure that the names you use in this
+section exactly match those in your permission manager.
+
+:::note
+If you have a group displayname set in LuckPerms, use that instead of the group name itself.
```

Comment:
I think we should mention somewhere that they are case sensitive.

Reply:
Although the phrase "the names you use in this section exactly match those in your permission manager" suggests this, I have included this piece of info on a follow-up commit to explicitly clarify this point. Thanks for pointing it out.